docs/publications.rst

```diff
@@ -90,6 +90,26 @@ please consider citing their respective papers as well:
       month = {March}
     }
 
+.. code-block:: tex
+
+    @inproceedings{drewniok2023quicksim_nano,
+	author    = {J. Drewniok and M. Walter and S. S. H. Ng and K. Walus and R. Wille},
+	title     = {{\emph{QuickSim}: Efficient \emph{and} Accurate Physical Simulation of Silicon Dangling Bond Logic}},
+	booktitle = nano,
```

Comment:
Please add the full name since this string is not defined.

Reply:
This has not been addressed yet.

Reply:
sorry, I thought you mean the ``domain``. I will change it immediately.